README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
```

Comment:
Highlight that the installation steps rely on Homebrew and assume the user is on macOS. It may be beneficial to provide alternative installation instructions or indicate this requirement for cross-platform usage.